custom_components/ramses_cc/services.yaml

```diff
@@ -133,33 +133,27 @@ set_system_mode:
             - custom
 
     period:
-      name: Days
+      name: Period (days)
```

Comment:
Renamed to make it clearer what this field is when switching between the UI/YAML and referencing descriptions.